### ISSUE TITLE
Next.js support

### ISSUE THREAD

**Issue by nandorojo** [intergalacticspacehighway/react-native-popper#5]
Hey there, I think there's some sort of issue with a dependency. I'm trying to import this in my Next.js app, and I'm seeing this:

<img width="954" alt="Screen Shot 2021-07-23 at 6 12 14 PM" src="https://user-images.githubusercontent.com/13172299/126846357-54f58fe0-cdff-459a-a0b1-7b31907d0333.png">

Any idea what that might be?

When running `yarn why dom-helpers`, I get this:

```sh
=> Found "dom-helpers@5.2.0"
info Reasons this module exists
   - "_project_#@beatgig#components#@material-ui#pickers#react-transition-group" depends on it
   - Hoisted from "_project_#@beatgig#components#@material-ui#pickers#react-transition-group#dom-helpers"
info Disk size without dependencies: "984KB"
info Disk size with unique dependencies: "2.76MB"
info Disk size with transitive dependencies: "2.8MB"
info Number of shared dependencies: 3
=> Found "@react-aria/overlays#dom-helpers@3.4.0"
info This module exists because "_project_#expo-next-app#react-native-popper#@react-native-aria#overlays#@react-aria#overlays" depends on it.
info Disk size without dependencies: "932KB"
info Disk size with unique dependencies: "1.6MB"
info Disk size with transitive dependencies: "1.64MB"
info Number of shared dependencies: 2
✨  Done in 1.34s.
```

**Comment by nandorojo** [comment]
Seems like the overlays library you're using is importing a version that isn't valid. I wonder if this could be solved with `yarn resolutions`.

**Comment by nandorojo** [comment]
this patch fixed it. probably worth solving with an explicit dependency version.

```diff
diff --git a/node_modules/@react-native-aria/overlays/lib/commonjs/web/overlays/src/calculatePosition.js b/node_modules/@react-native-aria/overlays/lib/commonjs/web/overlays/src/calculatePosition.js
index f90b89c..24a67bd 100644
--- a/node_modules/@react-native-aria/overlays/lib/commonjs/web/overlays/src/calculatePosition.js
+++ b/node_modules/@react-native-aria/overlays/lib/commonjs/web/overlays/src/calculatePosition.js
@@ -6,15 +6,15 @@ Object.defineProperty(exports, "__esModule", {
 exports.calculatePositionInternal = calculatePositionInternal;
 exports.calculatePosition = calculatePosition;
 
-var _style = _interopRequireDefault(require("dom-helpers/style"));
+var _style = _interopRequireDefault(require("dom-helpers/css"));
 
-var _offset = _interopRequireDefault(require("dom-helpers/query/offset"));
+var _offset = _interopRequireDefault(require("dom-helpers/offset"));
 
-var _position = _interopRequireDefault(require("dom-helpers/query/position"));
+var _position = _interopRequireDefault(require("dom-helpers/position"));
 
-var _scrollLeft = _interopRequireDefault(require("dom-helpers/query/scrollLeft"));
+var _scrollLeft = _interopRequireDefault(require("dom-helpers/scrollLeft"));
 
-var _scrollTop = _interopRequireDefault(require("dom-helpers/query/scrollTop"));
+var _scrollTop = _interopRequireDefault(require("dom-helpers/scrollTop"));
 
 var _ownerDocument = _interopRequireDefault(require("dom-helpers/ownerDocument"));
 
diff --git a/node_modules/@react-native-aria/overlays/lib/module/web/overlays/src/calculatePosition.js b/node_modules/@react-native-aria/overlays/lib/module/web/overlays/src/calculatePosition.js
index 8bb7bce..e81ebed 100644
--- a/node_modules/@react-native-aria/overlays/lib/module/web/overlays/src/calculatePosition.js
+++ b/node_modules/@react-native-aria/overlays/lib/module/web/overlays/src/calculatePosition.js
@@ -11,11 +11,11 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-import getCss from 'dom-helpers/style';
-import getOffset from 'dom-helpers/query/offset';
-import getPosition from 'dom-helpers/query/position';
-import getScrollLeft from 'dom-helpers/query/scrollLeft';
-import getScrollTop from 'dom-helpers/query/scrollTop';
+import getCss from 'dom-helpers/css';
+import getOffset from 'dom-helpers/offset';
+import getPosition from 'dom-helpers/position';
+import getScrollLeft from 'dom-helpers/scrollLeft';
+import getScrollTop from 'dom-helpers/scrollTop';
 import ownerDocument from 'dom-helpers/ownerDocument';
 const AXIS = {
   top: 'top',
```

**Comment by intergalacticspacehighway** [comment]
@nandorojo Thanks for reporting. This has been fixed in [0.2.0](https://github.com/intergalacticspacehighway/react-native-popper/releases/tag/v0.2.0). Let me know If you still face any issues!
